css/style.css

```diff
@@ -0,0 +1,442 @@
+    top: 10px;
+    transform: scale(1.2);
+}
+
+.current-date{
+	font-weight: 100;
+	color:#fff;
+}
+
+body{
+	background:#F4F5F6;
+	font-family: 'Roboto Condensed', sans-serif;
+}
+
+main{
+	padding:2rem;
+}
+
+.featured-stories-col img{
+	width: 100%;
+}
+
+a{
+	text-decoration:none;
+	color:#161616;
+}
+
+.posts-row{
+	max-width: 98%;
+    margin: auto;
+}
+
+.navbar{
+	border-bottom: 1px solid #e5e5e5;
+}
+
+
+.signin-btn{
+	background: transparent;
+	color:#fff;
+	font-weight: bold;
+}
+
+.subscribe-btn{
+	background: #fff;
+	border-radius: 0;
+	padding:.3rem 1.2rem;
+	font-weight: bold;
+	color:#f72210;
+}
+
+.p-topics{
+	margin-left: .7rem;
+    background: #afe2d2;
+    display: inline-block;
+    font-size:.87rem;
+    padding:.2rem .7rem;
+    font-family: sans-serif;
+}
+
+.p-short-desc{
+	font-weight: 300;
+    color: #666;
+    font-family: sans-serif;
+    margin:0;
+}
+
+.n-subscribe-btn{
+	border-radius: 0px;
+	margin:1rem 0;
+}
+
+.more-stories .highlights, .latest-posts .highlights{
+	padding-bottom: .5rem;
+	border-bottom: 1px solid #e5e5e5;
+}
+
+.more-stories .post, .latest-posts .post{
+	padding-top:.7rem;
+	border-bottom:1px solid #e5e5e5;
+	padding-bottom:.7rem;
+}
+
+.featured-story-col .post{
+  margin-bottom: 1rem;
+}
+
+.in-magazine,.editor-picks,.featured-slideshows{
+	margin-top: .8rem;
+}
+
+.in-magazine .top-title,.editor-picks .top-title, .featured-slideshows .top-title{
+	font-weight: bold;
+	color:#000;
+	border-bottom:1px solid #e5e5e5;
+	padding-bottom: .8rem;
+}
+
+.see-features{
+	background:#f4f5f6;
+	font-weight: bold;
+	color:#000;
+	padding:.5rem .8rem;
+}
+
+.in-magazine .top-title img,.editor-picks .top-title img, .featured-slideshows .top-title img{
+	max-width: 20px;
+    margin-right: .4rem;
+    position: relative;
+    top: -3px;
+}
+
+.cat-row{
+	border-bottom: 1px solid #e5e5e5;
+}
+
+.categories{
+	margin:4rem 0 2rem 0;
+	border-top:1px solid #e5e5e5;
+	padding:2rem 0;
+}
+
+.cat-title{
+	color: #f72210;
+	font-weight: bold;
+}
+
+.category-box .third-col .post{
+	padding-top:.7rem;
+	border-bottom:1px solid #e5e5e5;
+	padding-bottom:.7rem;
+}
+
+.nw-img{
+	height: 100%;
+	object-fit: cover;
+}
+
+.newsletter-row{
+	border:1px solid #e5e5e5;
+	max-width: 98%;
+	margin:auto;
+}
+
+.search-bar{
+	border:none;
+	background: transparent;
+	font-size: 1rem;
+}
+
+.fa-search{
+	position: absolute;
+	left: 50%;
+}
+
+.p-relative{
+	position: relative;
+}
+
+.highlights{
+	color:#666;
+}
+
+.post h4{
+  font-weight: bold;
+}
+
+
+.post h4, .post p{
+	color:#000;
+
+}
+
+.features{
+	background: #f72210;
+	color: #fff;
+	font-weight: bold;
+	padding:.3rem .8rem;
+	display: inline-block;
+}
+
+.p-category{
+	color:#f72210;
+	font-weight: bold;
+}
+
+
+
+.post h4:hover,.post h5:hover,.post h6:hover{
+	text-decoration: underline;
+	text-decoration-color: #f72210;
+	color:#000;
+}
+
+.post a{
+	text-decoration: none;
+}
+
+.post-img-frame{
+	position: relative;
+}
+
+.post-category{
+	position: absolute;
```

Comment:
`SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.